index.html

```diff
@@ -50,26 +52,30 @@
 	
 
 	<div id="editor">
-	<div id="files">
-		<span class="newFile" title="New File">+</span>
-	</div>
+		<div id="files">
+			<span class="newFile" title="New File">+</span>
+		</div>
 		<div id="input"></div>
+		<div id="dragbar"></div>
 	</div>
 
 	<div id="righthand-panel">
-		<div id="dragbar"></div>
 		<div id="header">
 			<img id="solIcon" src="solidity.svg">
 			<h1>Solidity realtime<br/>compiler and runtime</h1>
 			<div class="info">
 				<p>Version: <span id="version">(loading)</span><br/>
 				Change to: <select id="versionSelector"></select><br/>
-				Execution environment does not connect to any node, everyhing is local and in memory only.<br/>
+				Execution environment does not connect to any node by default, everything is local and in memory only.<br/>
```

Comment:
Can we change this text depending on the switch?
Or perhaps use this text as one of the two options in the switch, where the other is something like:
`Connect to node at localhost, transactions will be sent to the network and can cause loss of money or worse.`